docs/reference/scaffolding.md

```diff
@@ -0,0 +1,47 @@
+# Scaffolding
```

Comment:
Link this markdown in the `docs/reference/README.md`.

Reply:
Is now beeing linked in the README.md